### ISSUE TITLE
Question: Any of you have many err=255 on AP units?

### ISSUE THREAD

**Issue by mcalistair** [ael-code/daikin-control#10]
Hi "daikin-control" and other users,

First of all "daikin-control" thanks for all your work so far... Kudos.
@ All:
Since I got my 3 AP controllers:
1x BRP069A42 for FTXR42EV1B9 (singe Ururu Sarara)
2x BRP069A43 for FTXS20K2V1B (dual split unit)

I have "hangs/lockups" of the AP.
The time interval differs. Can be 1 week can be 3 weeks.
What happens is that the system will not react via the Daikin App and /common/basic_info will show err=255. 

Since today I found the /common/reboot call. This fixes the issue for me. I'm very happy as before I had to switch the AP's to standalone and back before they start working again.

My Question to you all: Do you have the same issues?
My local reseller + Daikin Netherlands + Europe cannot fix it, so I'm very curious.

PS the strange thing is that the AP is still working and accessible via the API but cannot communicate with the AirconUnit itself.

**Comment by StarkJohan** [comment]
Yep. Having the same issue once in a while, every couple of months or so. I normally just do a hard power off-on and it starts working again. I haven’t tried the reboot call but if that works it obviously is a better option. As I have the hard power off automatically programmed in my home automation system using a controllable wall plug I will just swap it to a reboot instead. Thanks for the tip.

Mvh
Johan Samuelsson
johan@snutt.net <mailto:johan@snutt.net>
+47 94 25 77 65

**Comment by nicdive** [comment]
Hi,

I have seven units and the exact same problem. I use reboot as well. Every unit fail sometime, it's really a pain in the a**. I tried several wifi AP and fixed the IP but no luck. 

Nicolas Buanec

**Comment by mcalistair** [comment]
Sounds strange but I'm a bit "glad" that I'm not the only one with this issue.
@StarkJohan --> good Idea I will also configure a rule in my HA (Vera) to call /common/reboot if needed.
@nicdive --> Be assured, it has nothing to do with your WiFi AP or settings. I traced the WiFi via logs and the issue is not in the WiFi it's in the communication between the BRP069A4X dongles and the AirCon unit itself.

**Comment by mcalistair** [comment]
All, FYI, Daikin released a new IOS app on 22nd of Feb 2016 (including a new FW release for the adapters 2.60). On of the items in the release notes states "WLAN Adapter: Improved Stability"
I Have updated all my 3 modules with this FW and they haven't went into ERR=255 yet.

**Comment by StarkJohan** [comment]
Awesome! Let’s hope it keeps working.

Now the only thing that I’m missing is starting “powerful” mode from the app/api...

**Comment by dide0** [comment]
The err= 255 or blinking power led has been solved with FW version 2.6.0. As with FW 2.2.5 only fix was to reboot your WalAN adapter.

With the new FW, the API has also slightly changed, so it could be that your current programes dont work anymore.

Regarding powerful mode, only units with PV: 3.2 or higher will support powerful (and econo) mode via the API. ( new 2016 R32 units).

**Comment by mcalistair** [comment]
@dide0 thats some neat inside info :+1:  (do you work at/for Daikin?)

**Comment by dide0** [comment]
@mcalistair  At Daikin, part of the smartphone App development team. I'm however not allowed to publish the API itself as it is not a public API, but on what was reverse engineered ... i could give some guidance.

**Comment by chaicka** [comment]
@dide0 
I believe many people around the world would benefit a great deal from Daikin API documentations if make available. In fact, it may help expand the interest and demand of these WiFi Adapters significantly, especially with the growing IoT market.

There are several variants of the adapters. I am using BRP072A42 for my models of fan coil.

**Comment by fredrike** [comment]
@mcalistair I saw that you are using BRP069A42  for a Ururu Sarara unit, what _extra_ options do you have in the original app? Can you for instance see power consumption, humidify the air and set fan mode to silent?

I just installed a BRP069A42 (that I bought on eBay) for my Ururu Sarara unit and can only see basic features in the *Daikin Online Controller* app and are afraid that I got a somewhat broken setup.

**Comment by mcalistair** [comment]
No extra options:
- see power consumption = NO
- humidify the air  = NO
- and set fan mode to silent = NO
Only Basic stuff is possible

So in other words I think its a limitation of the "early" type we have. It's not supported all those extra features. :(

**Comment by fredrike** [comment]
Sorry for hijacking the thread..

Ok, thank you @mcalistair, (I see that the current Sarara model comes with built-in WiFi,my unit is just 93 days old 😕 ). Will contact Daikin and see what they can do, as I can do all of the stuff via the remote control and it seems like it would be reasonable if the WiFi could do the same..

**Comment by dide0** [comment]
@fredrike As far as I remember, power consumption is only supported with the 2016+ R32 multi outdoor systems. I'll verify that

**Comment by fredrike** [comment]
@dide0 according to the Swedish site should the Ururu Sarara support both humidity settings and power consumption via the app..

https://www.daikin.se/sv_se/product-group/air-to-air-heat-pumps/r-serien/ururu_sarara.html

**Comment by dide0** [comment]
@fredrike , I've did some checks and the unit your refering to has an "old" protocol version, which doesn't communicate Consumption ( if you check /common/basic_info it should advertise "pv = 0" )

**Comment by pdionisis** [comment]
I also have an Ururu Sarara ftxz (bought recently) .
I recently bought also the BRP069**B**42 wifi module.I think it is the same as BRP069**A**42 but with 
US channels at 2.4Ghz band (only 1-11 channels instead of EU 1-13 channels)

Unfortunately I saw that I have access only to very basic control of my unit . On-off ,temperature and
heating-cooling.
I can not use it in practice, because I can not set "Humidify Heating" mode and can not set Fresh air, 
air purifing, wing direction etc.
So I can not use it. It is like if I have a very basic chinese unit with 300€ value , as I can not use any of the ururu sarara features.
 I am astonished that Daikin advertises (at the Ururu Sarara product page) that I can control the unit 
with online controller, while it seems that it is not compatible.

Dide0 what you said about "old" protocol version means that there is no hope of controlling our
unit remotely? (with all the ururu functions)
 Maybe we can update it? Or use another controller instead of BRP069A(B)042 ?

Somewhere I saw a project to use an arduino to send IR commands emulating the original IR control
but I did not find a library of IR commands for the extra ururu sarara commands.Any help?

Thank you

**Comment by apn3agr** [comment]
Wanted to revive this thread and check if there are any updates on this project. Kudos to those who initiated it. I am as frustrated as any ururu sarara customer, I was shocked with how basic the online support of the adaptor is
